Dockerfile

```diff
@@ -0,0 +1,42 @@
+# fog-sw BUILDER
+FROM ros:foxy-ros-base as fog-sw-builder
+
+ARG BUILD_NUMBER
+ARG COMMIT_ID
+ARG GIT_VER
+
+# workaround for ROS GPG Key Expiration Incident
+RUN rm /etc/apt/sources.list.d/ros2-latest.list && \
+    apt-get update && \
+    apt-get install -y curl && \
+    curl http://repo.ros2.org/repos.key | sudo apt-key add - && \
+    echo "deb http://packages.ros.org/ros2/ubuntu focal main" > /etc/apt/sources.list.d/ros2-latest.list && \
+    apt-get update
```

Comment:
this is not needed anymore and can be removed (fixed in base image)

Reply:
Removed.